test/integration/manifests/cns/daemonset-windows.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: azure-cns
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: azure-cns-win
+  namespace: kube-system
+  labels:
+    app: azure-cns-win
+spec:
+  selector:
+    matchLabels:
+      k8s-app: azure-cns-win
+  template:
+    metadata:
+      labels:
+        k8s-app: azure-cns-win
+        kubernetes.azure.com/managedby: aks
+      annotations:
+        cluster-autoscaler.kubernetes.io/daemonset-pod: "true"
+        prometheus.io/port: "10092"
+    spec:
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\SYSTEM"
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - operator: "Exists"
+          effect: NoExecute
+        - operator: "Exists"
+          effect: NoSchedule
+      containers:
+        - name: cns-container
+          image: acnpublic.azurecr.io/azure-cns:windows-amd64-ltsc2022-$WINDOWS_CNS_VERSION
```

Comment:
it should not be necessary to use the windows-specific tag. use the same multiplatform manifest tag that the linux tests use